src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Form/attachments_form_theme.html.twig

```diff
@@ -0,0 +1,92 @@
+{# **
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * #}
+{% extends 'PrestaShopBundle:Admin/TwigTemplateForm:prestashop_ui_kit_base.html.twig' %}
+
+{% block collection_widget %}
+    {%- block attached_file_row -%}
+      <tr class="attached_file_row" id="attached_file_row_{{ form.vars.name }}">
+        {% for child in form.children %}
+          {% set childType = child.vars.block_prefixes.1 %}
+          {% if childType != 'hidden' %}
+            <td>
+              {{ form_widget(child) }}
+              {% if block(child) %}
+                {{ form_errors(child) }}
+              {% endif %}
+            </td>
+          {% elseif child.vars.label is not empty %}
+            <td class="{{ child.vars.name }}">
+              {{ form_widget(child) }}
+              <span class="preview">{% if child.vars.value %}{{ child.vars.value }}{% endif %}</span>
+            </td>
+          {% else %}
+            {{ form_widget(child) }}
+          {% endif %}
+        {% endfor %}
+      </tr>
+    {%- endblock attached_file_row -%}
+
+    {%- block collection_row -%}
```

Comment:
I don't understand why customizing `collection_row` inside a `collection_widget`though...
Why not customizing `collection_row` only in this case? From what I understood only `attachements.attached_files` is a collection so only this one should need a custom theme